From 30c3d6ef2cb7a8b5f3cad99a3204a44560c06632 Mon Sep 17 00:00:00 2001
From: Suman Anna <s-anna@ti.com>
Date: Mon, 20 Jul 2015 15:44:56 -0500
Subject: [PATCH 1/8] TST: ARM: dts: OMAP3: Add iommu test device nodes

Add a iommu test device node on OMAP3 for testing. This
device node will probe a test driver which allows us to
perform a iommu unit test.

Signed-off-by: Suman Anna <s-anna@ti.com>
---
 arch/arm/boot/dts/omap3.dtsi | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/arch/arm/boot/dts/omap3.dtsi b/arch/arm/boot/dts/omap3.dtsi
index 4043ecb38016..bdc21923aad6 100644
--- a/arch/arm/boot/dts/omap3.dtsi
+++ b/arch/arm/boot/dts/omap3.dtsi
@@ -835,6 +835,12 @@
 			};
 		};
 	};
+
+	iommu_test {
+		compatible = "ti,omap-iommu-test";
+		iommus = <&mmu_isp>;
+		/*iommus = <&mmu_iva>;*/
+	};
 };
 
 /include/ "omap3xxx-clocks.dtsi"
-- 
2.22.0

